### PR TITLE
Add migration docs

### LINKED ISSUE #47: Add docs for Chassis migrations
We should add migration docs to assist with people who are new to Chassis and want to move a Chassis site from local to staging/production.

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #47 and #652.

### CODE REVIEW

File: docs/guides.rst [hunk 1]
```diff
@@ -5,6 +5,54 @@ Guides
...
 These guides will help you configure Chassis for specific types of development.
 
+Migrations
+----------
+
+Importing A Production Database Into Chassis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Provision a new Chassis instance.
+#. Export your production database with `WP-CLI`_ ``wp db export <filename.sql>``.
```
Comment (peterwilsoncc) [inline docs/guides.rst]: I've had problems with multi-byte characters using this command but can't remember the export command I used.

Cool story.

File: docs/guides.rst [hunk 2]
```diff
@@ -5,6 +5,54 @@ Guides
 
 These guides will help you configure Chassis for specific types of development.
 
+Migrations
+----------
+
+Importing A Production Database Into Chassis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Provision a new Chassis instance.
```
Comment (peterwilsoncc) [inline docs/guides.rst]:
```suggestion
#. Use ``ssh`` to connect to your production server. Your host should provide instructions for doing this.
```

File: docs/guides.rst [hunk 3]
```diff
@@ -5,6 +5,54 @@ Guides
...
+#. Export your production database with `WP-CLI`_ ``wp db export <filename.sql>``.
+#. Copy the export into the ``content`` folder.
```
Comment (peterwilsoncc) [inline docs/guides.rst]:
```suggestion
#. Download your the contents of your ``wp-content`` folder on production to your local content folder.
#. Provision a new Chassis instance.
#. Copy the export into the ``content`` folder.
```

File: docs/guides.rst [hunk 4]
```diff
@@ -5,6 +5,54 @@ Guides
...
+#. Run ``vagrant ssh`` in a terminal to SSH into your Chassis box.
+#. Run ``cd /vagrant/content``
+#. Run ``wp db import <filename.sql>``
+#. Run ``wp search-replace '//www.yoursite.com/wp-content' '//vagrant.local/content'``
+#. Run ``wp cache flush``
+#. Download your the contents of your ``wp-content`` folder on production to your local content folder.
```
Comment (peterwilsoncc) [inline docs/guides.rst]:
```suggestion
```

File: docs/guides.rst (hunk 1 above)
Comment (BronsonQuick) [reply docs/guides.rst]: Hmm. My guess is it was probably utf8mb4 going into utf8. It's an edge case so if someone hits it hopefully they'll open an issue and I can update the docs.

**Pull request merged**